puppet/modules/jenkins_master/manifests/init.pp

```diff
@@ -163,7 +157,6 @@
     'parameterized-trigger'              => {'version' => '2.35.2'},
     'pipeline-github-lib'                => {'version' => '1.0'},
     'pipeline-githubnotify-step'         => {'version' => '1.0.4'},
-    'xvfb'                               => {'version' => '1.1.3'},
```

Comment:
should have been removed in https://github.com/theforeman/foreman-infra/commit/5e241b7eff2ab6d522ad0fae2cd64031d07f8ad7 along with the changes in the two other files